deltachat-ios/ViewModel/ChatListViewModel.swift

```diff
@@ -158,6 +133,7 @@ class ChatListViewModel: NSObject, ChatListViewModelProtocol {
         return nil
     }
 
+    // needed to differentiate betweeen deaddrop / archive / default
```

Comment:
this comment is not so much adding new information (like about why) and the switch-case is quite easy to read in my opinion.

Reply:
you're right, i will remove it. i copied it down without thinking much about it :)